Magento_Theme/web/js/select.babel.js

```diff
@@ -1,25 +1,29 @@
 define(['choices'], function(Choices) {
-    'use strict';
+  'use strict';
 
-    return function(options, select) {
-        const defaultOptions = {
-            placeholder: false,
-            searchEnabled: false,
-            itemSelectText: '',
-            position: 'bottom',
-            sortFilter: (a, b) => {
-              return a.label.localeCompare(b.label);
-            },
-            classNames: {
-                containerInner: 'select__field',
-                list          : 'select__field-list',
-                input         : 'select__field-input',
-                item          : 'select__field-item',
-                listSingle    : 'select__field-list--single'
-            },
-            itemSelectText: ''
-        };
-        options = Object.assign(defaultOptions, options);
```

Comment:
default option are no more required?

Reply:
I made a small mistake here. Now I restored support for custom select options, they can be added this way:
```
data-mage-init='{
    "select": {
        "type": "number",
        "options": { "searchEnabled": "true" }
    }
}'
```